LEGO1/omni/include/mxdsaction.h

```diff
@@ -86,13 +86,13 @@ class MxDSAction : public MxDSObject {
 	void SetLoopCount(MxS32 p_loopCount) { m_loopCount = p_loopCount; }
 
 	// FUNCTION: BETA10 0x1003db50
-	Mx3DPointFloat& GetLocation() { return m_location; }
+	Vector3& GetLocation() { return m_location; }
```

Comment:
This change fixes a number of other functions in BETA10 which were supposed to use `Vector2::operator[]` but used `Mx3DPointFloat::operator[]` instead